metron-analytics/metron-profiler-common/src/main/java/org/apache/metron/profiler/hbase/SaltyRowKeyBuilder.java

```diff
@@ -81,20 +99,19 @@ public SaltyRowKeyBuilder(int saltDivisor, long duration, TimeUnit units) {
    * @return All of the row keys necessary to retrieve the profile measurements.
    */
   @Override
-  public List<byte[]> rowKeys(String profile, String entity, List<Object> groups, long start, long end) {
+  public List<byte[]> encode(String profile, String entity, List<Object> groups, long start, long end) {
     // be forgiving of out-of-order start and end times; order is critical to this algorithm
     end = Math.max(start, end);
     start = Math.min(start, end);
```

Comment:
This does look fishy.  I will open a separate JIRA and track this separately.  Thanks, @mattf-horton!